spec/lock_spec.rb

```diff
@@ -1,6 +1,7 @@
 require 'spec_helper'
 require 'json'
 require 'base64'
+require 'webmock/rspec'
```

Comment:
at this point, you could put all those requires in spec/spec_helper.rb and require it instead

Reply:
Done. Thanks